### ISSUE TITLE
Add ubuntu 24.04 to CI

### ISSUE THREAD

**Issue by gavv** [roc-streaming/roc-toolkit#634]
This needs two pull request:

1. Add `env-ubuntu:24.04` to [dockerfiles](https://github.com/roc-streaming/dockerfiles) repo; we need to add Dockerfile and images.csv entry [here](https://github.com/roc-streaming/dockerfiles/tree/main/images/env-ubuntu) similar to how it's done for other ubuntu versions

2. Add ubuntu 24.04 step to [build.yml](https://github.com/roc-streaming/roc-toolkit/blob/develop/.github/workflows/build.yml), again, similar to how it's done for other ubuntu versions

Basically, that's it, but if there will be CI failures on new version, we'll need to fix them.

Related docs:

* https://github.com/roc-streaming/roc-toolkit/blob/develop/docs/sphinx/development/continuous_integration.rst

**Comment by nolan-veed** [comment]
I'll do this.

**Comment by gavv** [comment]
Landed.
